src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -613,12 +604,12 @@ private MessageConsumer doBeginApplicationReply(
         {
             final String applicationReplyName = applicationName;
             final String peerHost = tlsEngine.getPeerHost();
```

Comment:
`peerHost` -> `tlsPeerHost` if using `tlsApplicationProtocol` naming convention for `applicationProtocol`.